stream_framework/feeds/aggregated_feed/base.py

```diff
@@ -274,6 +274,6 @@ def _translate_diff(self, new, changed, deleted):
         to_add = new[:]
         if changed:
             # sorry about the very python specific hack :)
```

Comment:
If you want to make this less hacky, something like `[c[0] for c in changed]` looks cleaner. If performance is a concern, profile it before making any change, though.